agreements.md

```diff
@@ -230,6 +230,15 @@ Thus, his available co-founding value returned to the original value of $512.
 As a senior Reef Technologies engineer, you are eligible to use the services of a personal assistant.
 The assistant may do the necessary research, make a list of top hardware models for you to choose from, find a good online shop that is available in your country and have the appropriate equipment delivered to your address.
 
+## LLM tool use & funding
+
+Each Reef Technologies senior staff member is eligible to receive company-funded LLM service subscription (e.g. GitHub Copilot, ChatGPT Plus).
+The subscription is paid by Reef Technologies for entire time employee is actively using it for any tasks related to Reef Technologies projects.
```

Comment:
due to the weird way in which PIP operates we tend to aviod the "employee" word as most of our staff is self-employed and it's important to keep it clear
```suggestion
The subscription is paid by Reef Technologies for entire time it is being used it for any tasks related to Reef Technologies projects.
Please make sure the subscription is terminated if you don't need it anymore to prevent license waste.
```